sdk/src/wallet/core/mod.rs

```diff
@@ -94,7 +94,7 @@ pub struct WalletInner<S: SecretManage = SecretManager> {
     #[cfg(feature = "storage")]
     pub(crate) storage_options: StorageOptions,
     #[cfg(feature = "storage")]
-    pub(crate) storage_manager: tokio::sync::RwLock<StorageManager>,
+    pub(crate) storage_manager: StorageManager,
```

Comment:
Why is it not needed anymore?

Reply:
All access is immutable. Figured it's better not to have the lock at all in that case. However, if you think there's a valid reason to leave it, let me know.